### ISSUE TITLE
Multiple Tag Fields

### ISSUE THREAD

**Issue by mwweb** [vdm-io/Joomla-Component-Builder#10]
I have created several tag fields for collecting data. I am wanting to re-purpose the Joomla! core as much as possible, and rather than trying to create something for comma separate values, then separate those in the code, Joomla! tags would be much cleaner.

I created several tag field, using your example, and have given each one a different name. In looking in admin/layouts/property (the name of my view), then looking at the PHP, in this case leatures_left.php, I can see that it looks like it's pulling everything correctly, except when it comes to where it places the tags. The PHP is showing:

	'tags',
	'tags',
	'tags',
	'fencing',
	'tags',
	'tags',

rather than pulling the name (or even ID) of the tag field.

![screenshot_fri_jul_08_18 09 17](https://cloud.githubusercontent.com/assets/19194012/16705321/8925b444-453a-11e6-93c9-946dd20f17c4.png)
![screenshot_fri_jul_08_18 09 45](https://cloud.githubusercontent.com/assets/19194012/16705320/8923d12e-453a-11e6-8bce-c2e5d1874221.png)

**Comment by Llewellynvdm** [comment]
Sorry but Joomla does not allow more then one tag field per view-table

**Comment by Llewellynvdm** [comment]
Wait let me rephrase, Component Builder does not allow for more then one tag due to the exstrem conventions. So I must say the tag field implementation is very tricky read this for some help:

https://docs.joomla.org/J3.x:Using_Tags_in_an_Extension
https://docs.joomla.org/J3.x:How_To_Use_Content_Tags_in_Joomla!

Then think about it, component build is trying to do all of that for you... so at this time it can only do one tag... sorry!
